src/main/java/uk/gov/hmcts/darts/event/service/handler/base/EventHandlerBase.java

```diff
@@ -69,6 +69,19 @@ protected CreatedHearingAndEvent createHearingAndSaveEvent(DartsEvent dartsEvent
             caseNumber,
             dartsEvent.getDateTime().toLocalDate()
         );
+        UserAccountEntity currentUser = authorisationApi.getCurrentUser();
```

Comment:
i actually implemented something similar in a recent ticket that is merged with master, might be worth double checking this is still needed.